spread.yaml

```diff
@@ -490,6 +490,10 @@ suites:
    summary: tests of snapcraft's craft-part's based plugins
    systems:
    - ubuntu-22.04*
+ tests/spread/plugins/craft-parts/colcon/:
```

Comment:
instead of this, can you layout the tests to follow the pattern on https://github.com/snapcore/snapcraft/tree/main/tests/spread/plugins/craft-parts/build-and-run-hello ?

Reply:
Done in https://github.com/snapcore/snapcraft/pull/3994/commits/6f689c8643241d135d15cc7652272588ee459be7